sso/src/main/java/dk/acto/fafnir/sso/provider/SamlProvider.java

```diff
@@ -75,6 +78,13 @@ public String getSamlRegistrationIds(String orgId) {
                 .orElseThrow(OrganisationNotUsingSaml::new);
     }
 
+    public List<OrganisationData> getOrgsForProvider() {
```

Comment:
Should this be public?

Reply:
It is used in https://github.com/actoaps/fafnir-sso/pull/86/files/20f1cc65c36f17ac5a37b713470f5b34a30b6a8c#diff-464769c558c7d4ec63d3d261f4bec8d1ccbb535a2924a88b248b76dc2a786fa1R47